WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -180,8 +201,10 @@ public final class WordPressComOAuthClient: NSObject {
                         failure(
                             NSError(
                                 domain: WordPressComOAuthClient.WordPressComOAuthErrorDomain,
-                                code: WordPressComOAuthError.noMultifactorHandlerGiven.rawValue,
-                                userInfo: nil
+                                code: WordPressComOAuthError.unknown.rawValue,
+                                userInfo: [
+                                    NSLocalizedDescriptionKey: "Response requires handling the MFA Webauthn flow but handler given ('needsMultifactor' parameter)."
+                                ]
```

Comment:
This assumes clients handle the error properly, e.g. by switching and reading the localized description.

I haven't verified the behavior. In any case, there is ongoing work to update WordPressAuthenticator (the only client for this method?) to support Webauthn for passkeys, which will address this occurrence.

See the details in the https://github.com/wordpress-mobile/WordPressKit-iOS/pull/629 description and pe5pgL-3HA-p2